charts/kong/values.yaml

```diff
@@ -398,7 +398,7 @@ postgresql:
 # -----------------------------------------------------------------------------
 
 waitImage:
-  repository: busybox
+  repository: centos
```

Comment:
Consider using `bash:5` instead of `centos:latest` because the former is 13.7MB and the latter is 125MB.

If you choose to go with centos though, I strongly recommend `centos:8` instead of `centos:latest` because major RHEL/CentOS releases come with surprising breaking changes normally.

Reply:
Sounds reasonable. I originally chose centos as probably the closest open-source public equivalent to the Red Hat UBI (which we use for the Red Hat certified stuff), but as we only care about bash and I don't expect differences in bash (fingers crossed on CI passing it), smaller image for that is good.

`latest` was a holdover from our busybox tag--it's probably more or less arbitrary since it seems unlikely that the bash component would change in a way that breaks this; I don't have particularly strong feelings either way for the basic utility image (if `latest` does end up breaking something for that, it's probably something we do need to investigate, because HOW!?), but as pinned is pretty much always the safer choice, seems reasonable to move to that.

Possible future security issues in an older image is the only concern, but bash vulnerabilities are the sort of thing that usually gets wide enough coverage that we hopefully shouldn't miss them.